openslides/motions/config_variables.py

```diff
@@ -237,4 +237,4 @@ def get_config_variables():
         label='Show paragraph numbering (only in PDF)',
```

Comment:
Hey this is easy. Just remove the if clauses in motions/pdf.py line 284 and 300 and let the always the else clauses run.